smartautoclicker/src/main/java/com/buzbuz/smartautoclicker/activity/PermissionsDialogFragment.kt

```diff
@@ -80,10 +80,6 @@ class PermissionsDialogFragment : DialogFragment() {
         return AlertDialog.Builder(requireContext())
```

Comment:
This should be in a dedicated pull request.
Also, when changing the behaviour of the application, it would be nice to explain the change (before/after).